문자열-내-p와-y의-개수.js

```diff
@@ -0,0 +1,13 @@
+function solution(s) {
+    var answer = false;
+    let counts = {
+        p: 0,
+        y: 0,
+    };
```

Comment:
객체는 안에 속성을 변경해도 객체 자체를 바꾸지 않으면 const를 사용하는게 좋을 거 같아요
```suggestion
    const counts = {
        p: 0,
        y: 0,
    };
```